common/scala/src/main/scala/whisk/core/entity/DefaultRuntimeVersions.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2015-2016 IBM Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package main.scala.whisk.core.entity
```

Comment:
Nice catch indeed!  I'm going to blame Eclipse for this one... that package was set by the New Trait wizard.  In fact, in Package Explorer, all my packages are shown starting with `main.scala`. Weird.

@rabbah Thanks for taking care of this 👍